tests/conftest.py

```diff
@@ -27,21 +27,21 @@ def client(app):
 
 @pytest.fixture(scope="session")
 def db():
-    db = SQLAlchemy(flask_app)
-    db.init_app(flask_app)
+    db = SQLAlchemy()
```

Comment:
Previously It was throwing an error while running unit test, that the SQLAlchemy instance was already defined